drivers/clock_control/clock_control_rpi_pico.c

```diff
@@ -102,7 +102,9 @@
 #define AUXSRC_clk_sys   CLK_SYS
 #define AUXSRC_clk_usb   CLK_USB
 #define AUXSRC_clk_adc   CLK_ADC
+#if defined(CONFIG_SOC_SERIES_RP20XX)
```

Comment:
Here, it would be a good idea to branch depending on whether clk_rtc exists or not.